vizro-ai/src/vizro_ai/chains/_llm_models.py

```diff
@@ -1,47 +1,30 @@
 from typing import Dict, Optional, Union
 
-from langchain_openai import ChatOpenAI
+from langchain_openai import AzureOpenAI, ChatOpenAI
 
 # TODO add new wrappers in if new model support is added
-LLM_MODELS = Union[ChatOpenAI]
-
-# TODO constant of model inventory, can be converted to yaml and link to docs
-PREDEFINED_MODELS: Dict[str, Dict[str, Union[int, LLM_MODELS]]] = {
-    "gpt-3.5-turbo-0613": {
-        "max_tokens": 4096,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-4-0613": {
-        "max_tokens": 8192,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-3.5-turbo-1106": {
-        "max_tokens": 16385,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-4-1106-preview": {
-        "max_tokens": 128000,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-3.5-turbo-0125": {
-        "max_tokens": 16385,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-3.5-turbo": {
-        "max_tokens": 16385,
-        "wrapper": ChatOpenAI,
-    },
-    "gpt-4-turbo": {
-        "max_tokens": 128000,
-        "wrapper": ChatOpenAI,
-    },
+LLM_MODELS = type(Union[ChatOpenAI, AzureOpenAI])  # Need type here to pass py3.9 unit test?
+
+SUPPORTED_MODELS = {
+    "OpenAI": [
+        "gpt-3.5-turbo-0613",
```

Comment:
This one is deprecated: https://platform.openai.com/docs/deprecations/2023-11-06-chat-model-updates